docs/reference/cli-commands/notice.md

```diff
@@ -18,3 +18,42 @@ by unique type and key combination (2-arg variant).
       --uid=            Look up notice from user with this UID (admin only; 2-arg variant only)
 ```
 <!-- END AUTOMATED OUTPUT -->
+
+## Examples
+
+You can fetch a notice either by ID or by type/key combination.
+
+ To fetch the notice with ID "1":
+
+```{terminal}
+   :input: pebble notice 1
+id: "1"
+user-id: 1000
+type: custom
+key: example.com/foo
+first-occurred: 2023-09-15T04:16:09.179395298Z
+last-occurred: 2023-09-15T04:16:19.487035209Z
+last-repeated: 2023-09-15T04:16:09.179395298Z
+occurrences: 3
+expire-after: 168h0m0s
+```
+
+To fetch the notice with type "custom" and key "other.com<span></span>/bar":
```

Comment:
```suggestion
To fetch the notice with type "custom" and key "other.com/bar":
```

Reply:
This is per Anne's comment and the Canonical style guide. The raw text looks weird but it prevents the key from being rendered as a link because it isn't a link.

Reply:
Hi @benhoyt , 

Yeah it isn't pretty but this seems to be the way to avoid getting it rendered.
https://canonical-documentation-with-sphinx-and-readthedocscom.readthedocs-hosted.com/style-guide-myst/#external-links

Or to convert it to inline code instead of being wrapped with double quotes.

Reply:
Oh, sorry -- I'd missed that. And while we're here, Tony would probably remind us that we should use `example.com` as the example domain. (other.com is an actual website)